### PR TITLE
fix(QTM-415):  Update ts file with new property to hide last page

### PR DESCRIPTION
**Pull request by MarcosViniciusPC**
## Description
https://jirasoftware.catho.com.br/browse/QTM-415

## Review guide
- [ ] Unit tests (yarn test:components)
- [ ] Regression
- [ ] Code review
- [ ] TypeScript updated

### CODE REVIEW

File: components/Pagination/utils/pagination.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ const pagination = ({
   if (penultimatePage < lastPage - 2) {
     visiblePages = [...visiblePages.slice(0, -1), '...'];
 
-    if (showLastPagination) {
+    if (!hideLastPagination) {
       visiblePages = [...visiblePages, lastPage];
```
Comment (douglasjr97) [inline components/Pagination/utils/pagination.js:75]:
```suggestion
    if (hideLastPagination) {
      visiblePages = [...visiblePages, lastPage];
```

you should adjust the logic with a hideLastPagination being true.

**Review by douglasjr97 — CHANGES_REQUESTED** [review]
you should adjust the logic.
 
image 

image

**Review by douglasjr97 — APPROVED** [review]
## Review guide
- [x] Unit tests (yarn test:components)
- [x] Regression
- [x] Code review
- [x] TypeScript updated

**Comment by MarcosViniciusPC** [comment]
> shouldn't it start in hiding?

No, the default behavior is always visible.

**Pull request merged**